src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state) => {
```

Comment:
- [ ] Your `addBook` reducer should have both a `state` and an `action`. At the moment, it is not having an `action`. Also, it not very necessary to initialize the properties of the book with initial values. Get a better view from the code snippet below:
```
addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },

```